app/utils.py

```diff
@@ -195,3 +197,24 @@ def is_classmethod(method, cls):
     with suppress(AttributeError, KeyError):
         return isinstance(cls.__dict__[method.__name__], classmethod)
     return False
+
+
+def try_download_template_email_file_from_s3(service_id, template_email_file_id):
+    file_path = f"{service_id}/{template_email_file_id}"
```

Comment:
Are we using this file naming convention because it's consistent with what we do for letters and letter attachments?

I don't really have any strong preferences about how we name it (so long as we have the file id in there) but just wanted to confirm the reasoning

Reply:
Yeah, I followed the pattern we use in `send_pdf_letter_notification()` 🙌🏼